repository.go

```diff
@@ -53,7 +65,19 @@ func NewRepository(path, password string) (Repository, error) {
 	return repository, err
 }
 
-// OpenRepository opens an existing repository
+// generateRandomKey generates a random key with a specific length
+func generateRandomKey(length int) (string, error) {
+	b := make([]byte, length)
+
+	_, err := rand.Read(b)
+	if err != nil {
+		return "", err
+	}
+
+	return base64.URLEncoding.EncodeToString(b), err
```

Comment:
We can be sure `err` is not nil here, so maybe it's a bit easier to parse/read to just return `..., nil` instead of `err` here.